errors/index.js

```diff
@@ -0,0 +1,15 @@
+exports.customErrors = (err, req, res, next) => {
+  if (err.status) {
+    res.status(err.status).send({ msg: err.msg });
+  } else next(err);
+};
+
+exports.psqlErrors = (err, req, res, next) => {
+  if (err.code === "22P02" || err.code === "23502") {
+    res.status(400).send({ msg: "bad request" });
+  } else next(err);
+};
+
```

Comment:
Really great error handling middleware functions, awesome work
However... you're not handling any errors in this ticket and you won't have come across any of the errors these functions are catching yet... so these are unnecessary until you start error handling.

Reply:
OK